app/StateMachines/SiteStatusStateMachine.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\StateMachines;
+
+class SiteStatusStateMachine extends EntityStatusStateMachine
+{
+    public const RESTORATION_IN_PROGRESS = 'restoration-in-progress';
+
+    public function transitions(): array
+    {
+        return [
+            self::STARTED => [self::AWAITING_APPROVAL],
+            self::AWAITING_APPROVAL => [self::APPROVED, self::NEEDS_MORE_INFORMATION],
+            self::NEEDS_MORE_INFORMATION => [self::APPROVED, self::RESTORATION_IN_PROGRESS],
```

Comment:
This needs to continue to include `AWAITING_APPROVAL`. I think it could just be added here, but perhaps better way to accomplish what's being added here would be to supplement the super's transitions (example taken from `ReportStatusStateMachine`, which does something similar)
```
public function transitions(): array
{
    $parentTransitions = parent::transitions();
    // Reports can go from awaiting approval to started if the nothing_to_report flag is true (see validations below)
    $parentTransitions[self::NEEDS_MORE_INFORMATION][] = self::RESTORATION_IN_PROGRESS;

    return array_merge(
        [
            self::RESTORATION_IN_PROGRESS => [self::NEEDS_MORE_INFORMATION, self::APPROVED],
        ],
        $parentTransitions
    );
}
```